packages/loader/container-loader/src/container.ts

```diff
@@ -613,6 +613,9 @@ export class Container extends EventEmitterWithErrorHandling<IContainerEvents> i
                         this.propagateConnectionState();
                     }
                 },
+                onCaughUpToKnownOps: (callback: () => void) => {
```

Comment:
The signature you added above for "callback" is different.

Reply:
Yeah this code doesn't care about the boolean parameter in the callback so just omit it.  The function types are compatible (the argument will just be dropped by JS).